### PR TITLE
Implement UI for Home Screen

### PR DESCRIPTION
**Pull request by SashaBataieva**
UI for initiating connection to grid server #6

### CODE REVIEW

File: Example/SwiftSyft/HomeViewController.swift [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+    @IBOutlet weak var sendButton: UIButton!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        headerDescriptionLabel.text = StaticHomeScreenStrings.headerDescription
+
+        descriptionTextView.text = StaticHomeScreenStrings.description
+        descriptionTextView.attributedText = descriptionTextView.attributedText?
+            .fillInLink(StaticHomeScreenStrings.swiftSyftKey, with: StaticHomeScreenStrings.swiftSyft, url: StaticHomeScreenStrings.swiftSyftURL)
+            .fillInLink(StaticHomeScreenStrings.openMinedKey, with: StaticHomeScreenStrings.openMined, url: StaticHomeScreenStrings.openMinedURL)
```
Comment (mjjimenez) [inline Example/SwiftSyft/HomeViewController.swift]: This is minor, but there's a line length violation triggered on these lines on swiftlint. You can see the warnings swiftlint generates after you build and warnings will appear on the issue navigator of Xcode (Command + 5). You can just create a newline after the parameters like this:

```
            .fillInLink(StaticHomeScreenStrings.swiftSyftKey, with: StaticHomeScreenStrings.swiftSyft, 
                                                               url: StaticHomeScreenStrings.swiftSyftURL)
```

Swiftlint line length violations can be annoying so we can review if you guys want to modify the minimum line length in the future.

File: Example/SwiftSyft/HomeViewController.swift [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+    static let description = """
+            This is a demo using @SwiftSyft@ from @OpenMined@ to execute a multi-worker protocol hosted on grid.js
+    """
+    static let swiftSyft = "SwiftSyft"
+    static let swiftSyftKey = "@SwiftSyft@"
+    static let swiftSyftURL = "https://github.com/OpenMined/SwiftSyft"
+    static let openMined = "OpenMined"
+    static let openMinedKey = "@OpenMined@"
+    static let openMinedURL = "https://github.com/OpenMined/grid.js"
+    static let socketURL = "ws://localhost:3000"
```
Comment (mjjimenez) [inline Example/SwiftSyft/HomeViewController.swift]: Let's change "ws://localhost:300" to "wss://localhost:300" to convey that we want the user to use a secure protocol.

Comment (SashaBataieva) [reply Example/SwiftSyft/HomeViewController.swift]: We will use "wss://localhost:300" (I mean local host 300 not 3000) ?

File: Example/SwiftSyft/HomeViewController.swift (hunk 1 above)
Comment (SashaBataieva) [reply Example/SwiftSyft/HomeViewController.swift]: ok, I will test the keyboard behavior on a device and will fix it. @mjjimenez Thank you :)

File: Example/SwiftSyft/HomeViewController.swift (hunk 2 above)
Comment (mjjimenez) [reply Example/SwiftSyft/HomeViewController.swift]: Oops, I meant 3000.

**Comment by mjjimenez** [comment]
Looks good. There's a line length violation on line 146, but I feel like the default setting is too unforgiving. I'll change it to 180 in a new pull request.

The travis build is failing because of some configuration issues but it's building fine on my side. I'll file another issue for me to fix it.

**Pull request merged**